voting/src/general_meeting.rs

```diff
@@ -35,6 +35,7 @@ enum ExecuteError {
     TallyingTimePassed,
     TallyingIsBeforeVotingEnd,
     VotingEndPassed,
+    InvalidAdmin,
```

Comment:
Actually admin is not invalid. The transaction signer is not admin.
How about `NotAuthorized` or `NotAdmin`?